pkg/sql/backfill.go

```diff
@@ -566,7 +566,7 @@ func (sc *SchemaChanger) validateConstraints(
 	return grp.Wait()
 }
 
-// getTableVersion retrieves the descriptor for the table being
+// GetTableVersion retrieves the descriptor for the table being
```

Comment:
comment mismatch, GetTableVersion is not yet uppercase in commit "export symbols from table collection"

Reply:
Done.